test/action.test.js

```diff
@@ -0,0 +1,200 @@
+})
+
+tap.test('should process dependabot PR and skip PR not in target', async t => {
+  const PR_NUMBER = Math.random()
+  const { action, stubs } = buildStubbedAction({
+    payload: {
+      pull_request: {
+        user: {
+          login: BOT_NAME
+        },
+        number: PR_NUMBER,
+        title: 'bump foo from 3.18.0 to 4.0.0'
+      }
+    },
+    inputs: { PR_NUMBER, TARGET: 'minor' }
+  })
+
+  await action()
+
+  t.ok(stubs.logStub.logWarning.calledOnceWith('Target specified does not match to PR, skipping.'))
+  t.ok(stubs.approveStub.notCalled)
+  t.ok(stubs.mergeStub.notCalled)
+})
+
+tap.test('should process test PR and skip PR not in target', async t => {
+  const PR_NUMBER = Math.random()
+  const { action, stubs } = buildStubbedAction({
+    payload: {
+      pull_request: {
+        number: PR_NUMBER,
+        user: { login: 'foo-bar' },
+        title: 'bump foo from 3.18.0 to 4.0.0'
+      }
+    },
+    inputs: { PR_NUMBER, TARGET: 'minor' }
+  })
+
+  process.env.NODE_ENV = 'test-ga-action'
+  await action()
+  process.env.NODE_ENV = undefined
+
+  t.ok(stubs.logStub.logWarning.calledOnceWith('Target specified does not match to PR, skipping.'))
+  t.ok(stubs.approveStub.notCalled)
+  t.ok(stubs.mergeStub.notCalled)
+})
+
+tap.test('should ignore excluded package', async t => {
+  const PR_NUMBER = Math.random()
+  const { action, stubs } = buildStubbedAction({
+    payload: {
+      pull_request: {
+        number: PR_NUMBER,
+        user: { login: BOT_NAME },
+        head: { ref: 'dependabot/npm_and_yarn/foo-0.0.1' },
+      }
+    },
+    inputs: { PR_NUMBER, TARGET: 'any', EXCLUDE_PKGS: ['foo'] }
+  })
+
+  await action()
+
+  t.ok(stubs.logStub.logInfo.calledOnceWith('foo is excluded, skipping.'))
+  t.ok(stubs.approveStub.notCalled)
+  t.ok(stubs.mergeStub.notCalled)
+})
+
+tap.test('apply only should not merge', async t => {
+  const PR_NUMBER = Math.random()
+  const { action, stubs } = buildStubbedAction({
+    payload: { issue: {} },
+    inputs: { PR_NUMBER, TARGET: 'any', EXCLUDE_PKGS: [], APPROVE_ONLY: true }
+  })
+
+  stubs.prStub.resolves({
+    number: PR_NUMBER,
+    user: { login: BOT_NAME },
+    head: { ref: 'dependabot/npm_and_yarn/foo-0.0.1' },
+  })
+
+  stubs.approveStub.resolves({ data: true })
+
+  await action()
+
+  t.ok(stubs.logStub.logInfo.calledOnceWith('Approving only'))
+  t.ok(stubs.mergeStub.notCalled)
+})
+
+tap.test('should review and merge', async t => {
+  const PR_NUMBER = Math.random()
+  const { action, stubs } = buildStubbedAction({
+    payload: {
+      pull_request: {
+        number: PR_NUMBER,
+        user: { login: BOT_NAME },
+        head: { ref: 'dependabot/npm_and_yarn/foo-0.0.1' },
+      }
+    },
+    inputs: { PR_NUMBER, TARGET: 'any', EXCLUDE_PKGS: [] }
+  })
+
+  await action()
+
+  t.ok(stubs.logStub.logInfo.calledOnceWith('Dependabot merge completed'))
+  t.ok(stubs.approveStub.calledOnce)
+  t.ok(stubs.mergeStub.calledOnce)
+})
```

Comment:
new line

Reply:
added `eol-last` rule to the `eslint` config